packages/client/tests/e2e/_utils/standard.cmd.sh

```diff
@@ -3,16 +3,16 @@
 set -euo pipefail
 
 # Get access to localhost
-export DOCKER_BRIDGE_IP=$(ip route | grep default | awk '{print $3}') # 172.17.0.1
+export DOCKER_BRIDGE_HOST="host.docker.internal"
```

Comment:
This change makes host access work correctly on both Mac and Linux. Probably Windows too, but I ahve not tested that